src/test/regress/expected/isolation_add_node_vs_reference_table_operations.out

```diff
@@ -7,35 +7,36 @@ create_distributed_table
 (1 row)
 
 step s1-do-not-replicate-on-activate:
- SET citus.replicate_reference_tables_on_activate TO off;
+	SET citus.replicate_reference_tables_on_activate TO off;
 
 step s2-load-metadata-cache:
- COPY test_reference_table FROM PROGRAM 'echo 1 && echo 2 && echo 3 && echo 4 && echo 5';
+	COPY test_reference_table FROM PROGRAM 'echo 1 && echo 2 && echo 3 && echo 4 && echo 5';
 
 step s1-begin:
     BEGIN;
 
 step s1-add-second-worker:
- SELECT 1 FROM master_add_node('localhost', 57638);
+	SELECT 1 FROM master_add_node('localhost', 57638);
 
 ?column?
 ---------------------------------------------------------------------
        1
 (1 row)
 
 step s2-copy-to-reference-table:
- COPY test_reference_table FROM PROGRAM 'echo 1 && echo 2 && echo 3 && echo 4 && echo 5';
-
-step s1-commit:
+	COPY test_reference_table FROM PROGRAM 'echo 1 && echo 2 && echo 3 && echo 4 && echo 5';
+ <waiting ...>
```

Comment:
See https://github.com/citusdata/citus/pull/5912#discussion_r873528381 for these changes